ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -1641,28 +1645,26 @@ public int compareTo(CompressedOwid other) {
      * Check if the delete delta folder needs to be scanned for a given split's min/max write ids.
      *
      * @param orcSplitMinMaxWriteIds
-     * @param deleteDeltaDir
+     * @param deleteDelta
+     * @param stmtId statementId of the deleteDelta if present
      * @return true when  delete delta dir has to be scanned.
      */
     @VisibleForTesting
     protected static boolean isQualifiedDeleteDeltaForSplit(AcidOutputFormat.Options orcSplitMinMaxWriteIds,
-        Path deleteDeltaDir)
-    {
-      AcidUtils.ParsedDelta deleteDelta = AcidUtils.parsedDelta(deleteDeltaDir, false);
+        AcidInputFormat.DeltaMetaData deleteDelta, Integer stmtId) {
```

Comment:
nit: extra spaces?

Reply:
it is the second line of parameters, no extra space here

Reply:
:)